ui/src/app/app-routing.module.ts

```diff
@@ -15,14 +15,14 @@
 
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import {HomeComponent} from "./components/home/home.component";
-import {LoginComponent} from "./components/login/login.component";
-import {WorkflowsComponent} from "./components/workflows/workflows.component";
-import {RunsComponent} from "./components/runs/runs.component";
-import {AuthGuardService} from "./services/guards/authGuard.service";
-import {LogInGuardService} from "./services/guards/logInGuard.service";
+import {HomeComponent} from './components/home/home.component';
+import {LoginComponent} from './components/login/login.component';
+import {WorkflowsComponent} from './components/workflows/workflows.component';
+import {RunsComponent} from './components/runs/runs.component';
+import {AuthGuardService} from './services/guards/authGuard.service';
+import {LogInGuardService} from './services/guards/logInGuard.service';
 import {routeNames} from './constants/routes.constants';
-import {WorkflowsHomeComponent} from "./components/workflows/workflows-home/workflows-home.component";
+import {WorkflowsHomeComponent} from './components/workflows/workflows-home/workflows-home.component';
```

Comment:
Afaik, there is no technical argument for preferring either double-quotes or single quotes. One could make the argument to favor single quotes because it requires only one keystroke one a english keyboard while double quotes take to, but imo, that doesn't matter too much.
Eventually we will have to solve IDE auto-formatting problems by configuring an auto-formatter. For now I won't fuss much about it if your IDE does it automatically.

Reply:
Thanks, I would like to keep it as single quotes.